canonical-websites/www.ubuntu.com
# 2659: Move /about/ubuntu-and-debian to /community/debian

# Note
**Please merge #2685 first**

## Done

- Moved /about/about-ubuntu/ubuntu-and-debian to /community/canonical
- Updated copy according to [copydoc](https://docs.google.com/document/d/16wyBJyzmRJMkz677QFkfXWapie22SyGrjXHduo8GibA/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/community/debian
- Check that it matches the [copydoc](https://docs.google.com/document/d/16wyBJyzmRJMkz677QFkfXWapie22SyGrjXHduo8GibA/edit#)
- Check that /about/about-ubuntu/ubuntu-and-debian redirects to /community/debian

## Issue / Card

Fixes #2659
